packages/cli/src/commands/replay/replay.command.ts

```diff
@@ -13,6 +13,7 @@ import {
   ReplayTarget,
   StoryboardOptions,
 } from "@alwaysmeticulous/common";
+import * as Sentry from "@sentry/node";
```

Comment:
[nit, not important, feel free to ignore] I think it can sometimes help tree shaking, and also make it clearer to human readers which parts of Sentry we actually depend upon if you do:

```suggestion
import { startTransaction } from "@sentry/node";
```